opm/simulators/linalg/PreconditionerWithUpdate.hpp

```diff
@@ -77,14 +77,70 @@ class DummyUpdatePreconditioner : public PreconditionerWithUpdate<typename Origi
     OriginalPreconditioner orig_precond_;
 };
 
-
 template <class OriginalPreconditioner, class... Args>
 std::shared_ptr<DummyUpdatePreconditioner<OriginalPreconditioner>>
-wrapPreconditioner(Args&&... args)
+getDummyUpdateWrapper(Args&&... args)
 {
     return std::make_shared<DummyUpdatePreconditioner<OriginalPreconditioner>>(std::forward<Args>(args)...);
 }
 
+template <class OriginalPreconditioner, class MatrixPtr>
```

Comment:
Why do you stora a matrix pointer, and not a const reference?

Reply:
changed.

Reply:
Name of type is still `MatrixPtr`... (as well as variable name)

Reply:
Template name and variable name now updated.